xbmc/windowing/gbm/DRMUtils.h

```diff
@@ -89,6 +89,9 @@ class CDRMUtils
   virtual bool AddProperty(struct drm_object *object, const char *name, uint64_t value) { return false; }
   virtual bool SetProperty(struct drm_object *object, const char *name, uint64_t value) { return false; }
 
+  static uint32_t ToAlpha(uint32_t fourcc);
+  static uint32_t FromAlpha(uint32_t fourcc);
```

Comment:
Not a big fan of the names, maybe `FourCCWithAlpha`/`FourCCWithoutAlpha`? (just a suggestion)

Reply:
fine by me